### ISSUE TITLE
Mobs spawning in unreachable air bubbles

### ISSUE THREAD

**Issue by ImreBoersma** [Iskallia/Vault-public-S1#80]
As shown below in this vault room, mobs can spawn in 'unreachable' air pockets.
Not a severe issue but weird nonetheless.

<img src="https://user-images.githubusercontent.com/22447422/112655540-c7782600-8e50-11eb-8665-09bf4bb1e08b.png" width="50%"/>
<img src="https://user-images.githubusercontent.com/22447422/112655546-c941e980-8e50-11eb-85d3-89bde16f7bb1.png" width="50%"/>

**Comment by ImreBoersma** [comment]
Also found this in other rooms:
<img src="https://user-images.githubusercontent.com/22447422/112661810-36587d80-8e57-11eb-867f-563628fcc677.png" width="50%"/>
<img src="<img src="https://user-images.githubusercontent.com/22447422/112655546-c941e980-8e50-11eb-85d3-89bde16f7bb1.png" width="50%"/>" width="50%"/>

Sorry if this is just nitpicking, but I thought I'd report it over here

**Comment by bmemike** [comment]
This is true for vanilla as well. A mob can spawn in any spawnable location - and dynamic terrain can result in isolated pockets where mobs spawn.
